### ISSUE TITLE
Warning: Invalid configuration encountered

### ISSUE THREAD

**Issue by chippyash** [RafPe/serverless-reqvalidator-plugin#28]
Warning: Invalid configuration encountered                                                                                                                      [15/1523]
  at 'functions.plan.events.0.http.reqValidatorName': must be string                                                                                                     
  at 'functions.customer.events.0.http.reqValidatorName': must be string                                                                                                 
  at 'functions.customer.events.2.http.reqValidatorName': must be string                                                                                                 
  at 'functions.user.events.0.http.reqValidatorName': must be string 

My serverless def looks like:
```yaml
reqValidatorName:
      Fn::ImportValue: 'apiGwRequestValidateBody'
```

I think I see this has occurred before in issue #16 . It's happening again. I'm using serverless V3.0.0  The app does build so it's not breaking yet.

Thanks for the plugin however :-)

**Comment by RafPe** [comment]
@chippyash can you specify the detailed version of both framework and plugin :) ?

**Comment by chippyash** [comment]
```bash
npx serverless --version
Framework Core: 3.3.0 (local)
Plugin: 6.1.0
SDK: 4.3.1
```
```bash
cat node_modules/serverless-reqvalidator-plugin/package.json
{
  "name": "serverless-reqvalidator-plugin",
  "version": "2.0.0",
  "description": "Serverless plugin for setting request validation",
  "main": "src/index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/RafPe/serverless-reqvalidator-plugin.git"
  },
  "keywords": [
    "serverless",
    "plugin",
    "requestvalidator"
  ],
  "author": "RafPe < me@rafpe.ninja >",
  "contributors": [
    "pj035 pm.jaecks@gmail.com (http://pjaecks.de)"
  ],
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/RafPe/serverless-reqvalidator-plugin/issues"
  },
  "homepage": "https://github.com/RafPe/serverless-reqvalidator-plugin#readme"
}
```

**Comment by RafPe** [comment]
Should be now all fixed via #31 Please re-test
